cmd/influx/cli/cli.go

```diff
@@ -396,7 +396,7 @@ func (c *CommandLine) clear(cmd string) {
 }
 
 func (c *CommandLine) use(cmd string) {
-	args := strings.Split(strings.TrimSuffix(strings.TrimSpace(cmd), ";"), " ")
+	args := strings.SplitAfterN(strings.TrimSuffix(strings.TrimSpace(cmd), ";"), " ", 2)
```

Comment:
This should probably be a `SplitN`, not a `SplitAfterN`. It doesn't really change anything in this case (N=2, and only care about the second item), but the separator isn't important after the split, which would be the only reason to use `SplitAfter`, since it keeps the returned strings.